public/js/contribute/pad/save.js

```diff
@@ -88,88 +91,124 @@ function retrieveItems(kwargs) {
     datum.has_content = datum.txt?.trim()?.length > 0;
     // items.push(datum);
     // SET THE fullTxt REPRESENTATION
-    if (store_instructions && datum.instruction) {
-      // text += `${datum.instruction}\n`;
-      text += `${sel.select('.instruction').node().innerText}\n`
+    let innerText = '';
```

Comment:
what does the innerText var do more than datum.has_content here? (and above)

Reply:
datum.txt.trim() can still be empty

Reply:
yes, you're right datum.txt cannot be empty here

I have the innerText changes everywhere in the same way to be consistent and make it easier to see whether the changes are the same everywhere